justfile

```
@@ -0,0 +1,253 @@
+# See https://just.systems/man/en
+
+# If DOCKER_REGISTRY is not already set, use a bogus registry with a unique
+# domain name so that it's virtually impossible to accidentally use an older
+# cached image.
+_test-id := `tr -dc 'a-z0-9' </dev/urandom | fold -w 5 | head -n 1`
+export DOCKER_REGISTRY := env_var_or_default("DOCKER_REGISTRY", "test-" + _test-id + ".local/linkerd")
+
+# The docker image tag.
+image-tag := `bin/root-tag`
+
+# The Linkerd CLI binary.
+linkerd := "bin/linkerd"
+
+_ctx := "--context=k3d-" + test-cluster-name
+_linkerd :=  linkerd + " " + _ctx
```

Little indent issue here

```suggestion
_linkerd := linkerd + " " + _ctx
```